src/sentry/models/search_common.py

```diff
@@ -8,3 +8,5 @@ class SearchType(IntEnum):
     REPLAY = 3
     METRIC = 4
     SPAN = 5
+    ERROR = 6
```

Comment:
Dang I see we use SearchType.EVENT in places that combine errors and transactions, otherwise we could have reused that